spirv_glsl.cpp

```diff
@@ -9852,7 +9972,7 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 	{
 		auto &type = get<SPIRType>(ops[0]);
 		if (type.vecsize > 1)
-			GLSL_UFOP(not );
+			GLSL_UFOP(not);
```

Comment:
Fallout from running formatting script?

Reply:
probably.

Reply:
@HansKristian-Work want anything done about it?